<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GarbageCenter</class>
 <widget class="QDialog" name="GarbageCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>서비스 고객 목록</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="LWCustomList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QStackedWidget" name="stackedWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="LEInputText"/>
       </item>
       <item>
        <widget class="QPushButton" name="BtnInput">
         <property name="text">
          <string>입력</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LEInputText</sender>
   <signal>returnPressed()</signal>
   <receiver>BtnInput</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
